### PR TITLE
`fk status`: Print keys and team members for memberships

### PR DESCRIPTION
**Pull request by idrysdale**
This PR handles all of a user's memberships, or rather all keys they are members of a team with.

If prints something that looks like this:
Screenshot 2019-03-21 at 19 04 28

### CODE REVIEW

File: fk/status.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+		printHeader(groupedMembership.Team.Name)
+
+		adminOfTeam := false
+
+		teamKeysWithWarnings := []table.KeyWithWarnings{}
+
+		for _, membership := range groupedMembership.Memberships {
+			key, err := loadPgpKey(membership.Me.Fingerprint)
+			if err != nil {
+				return 1
```
Comment (paulfurley) [inline fk/status.go]: error swallowed: should be `out.Print(ui.FormatFailure(..))`

File: fk/status.go [hunk 2]
```diff
@@ -18,10 +18,60 @@
...
+	"github.com/fluidkeys/fluidkeys/status"
+	"github.com/fluidkeys/fluidkeys/table"
 )
 
 func statusSubcommand(args docopt.Opts) exitCode {
+	out.Print("\n")
+
+	groupedMemberships, err := user.GroupedMemberships()
+	if err != nil {
+		log.Panic(err)
```
Comment (paulfurley) [inline fk/status.go]: eek

Comment (paulfurley) [reply fk/status.go]: i closed this card once! :P

Comment (paulfurley) [reply fk/status.go]: i've pushed a commit to fix this

File: fk/status.go [hunk 3]
```diff
@@ -85,6 +85,8 @@ func statusSubcommand(args docopt.Opts) exitCode {
 				colour.Cmd("fk team join " + groupedMembership.Team.UUID.String()),
 			}))
 		}
+
+		out.Print("To fetch and store team members keys run " + colour.Cmd("fk team fetch") + "\n\n")
```
Comment (paulfurley) [inline fk/status.go]: How about "The team's keys are updated automatically. To check for updates now, run fk team fetch"

Comment (idrysdale) [reply fk/status.go]: Sounds good to me!

**Comment by paulfurley** [comment]
I think `Never` would look better as either `never` or `-`

and `Admin` would look better as `admin`

**Comment by idrysdale** [comment]
Contenious, but you can have it 😉

**Comment by paulfurley** [comment]
AWESOME

**Pull request merged**